packages/components/src/components/bal-input/index.html

```diff
@@ -46,21 +40,35 @@ <h2 class="title is-size-2">Inverted</h2>
 
       <h2 class="title is-size-2">Number Input</h2>
       <section>
-        <bal-input id="bal-input-only-numbers" number-keyboard name="age" placeholder="Enter your age"></bal-input>
-        <p class="help" id="bal-input-only-numbers-preview"></p>
+        <p>With number-input property your can only enter valid numbers, if you start with dot (.) zero will be automatically added in front of it.</p>
```

Comment:
place the p elements before the section, because everything in the section will be shown in the docu as and example ;-)